source/_patterns/typography/examples.hbs

```diff
@@ -21,9 +21,27 @@
 			<h6 data-variant="light">Headline 6</h6>
 		</db-tab>
 		<db-tab label="Body" name="tab-bar">
+			<h1>functional</h1>
 			<p>Primary font: "DB Screen Sans", "Helvetica", "Arial", sans-serif;</p>
 			<p><em>Primary font italic: "DB Screen Sans", "Helvetica", "Arial", sans-serif;</em></p>
 			<p><strong>Primary font bold: "DB Screen Sans", "Helvetica", "Arial", sans-serif;</strong></p>
+			<p class="DO-NOT-COPY-THIS-CLASS-example-typography-tiny">I'm Tiny Tina</p>
```

Comment:
;) should we change this example text before merging?

Reply:
fixed :'(